### ISSUE TITLE
[findrepe] FileNotFoundException causing FindRepe to exit

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jareens/softenido#15]
```
I ran FindRepe via the following command on windows XP pro:

java -jar FindRepe.jar -w 1m "C:\Dokumente und Einstellungen\marians\Eigene 
Dateien" > filelist.txt

After a while I got a bunch of exceptions and the program halted with this 
output:

09.07.2010 10:12:44 com.softenido.cafe.io.FileHash equalsHash
SCHWERWIEGEND: null
java.io.FileNotFoundException: C:\Dokumente und Einstellungen\marians\Eigene 
Dateien\htdocs\_blog\wp-content\uploads\2006\06\markenprofile\MarkenProfile_10_K
osmetik.pdf (Zugriff verweigert)
        at java.io.RandomAccessFile.open(Native Method)
        at java.io.RandomAccessFile.<init>(RandomAccessFile.java:212)
        at com.softenido.cafe.util.FileDigest.buildHash(FileDigest.java:128)
        at com.softenido.cafe.util.FileDigest.getHash(FileDigest.java:116)
        at com.softenido.cafe.io.FileHash.equalsHash(FileHash.java:321)
        at com.softenido.cafe.io.FileHash.equals(FileHash.java:111)
        at java.util.HashMap.getEntry(HashMap.java:349)
        at java.util.LinkedHashMap.get(LinkedHashMap.java:280)
        at com.softenido.cafe.util.ArrayUtils.splitEquals(ArrayUtils.java:115)
        at com.softenido.cafe.util.ArrayUtils.splitEquals(ArrayUtils.java:136)
        at com.softenido.findrepe.FindRepePipe$8.filter(FindRepePipe.java:311)
        at com.softenido.findrepe.FindRepePipe$8.filter(FindRepePipe.java:307)
        at com.softenido.cafe.util.concurrent.ASyncFilter.call(ASyncFilter.java:47)
        at com.softenido.cafe.util.concurrent.ASyncValue.run(ASyncValue.java:60)
        at com.softenido.cafe.util.concurrent.actor.ActorSingle.run(ActorSingle.java:113)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:619)


Here is a german-english-glossary:

"SCHWERWIEGEND" means "FATAL".
"Zugriff verweigert" means "Access denied".

I tried changing the file name of the file "C:\Dokumente und 
Einstellungen\marians\Eigene 
Dateien\htdocs\_blog\wp-content\uploads\2006\06\markenprofile\MarkenProfile_10_K
osmetik.pdf" via the windows explorer and got an "access denied" error message 
as well. I could however delete the file.

I don't know if that's possible, but it would be great if FindRepe could handle 
this exception more gracefully, so that it would simply ignore files like this.

Thanks!
```

Original issue reported on code.google.com by `marian.s...@gmail.com` on 9 Jul 2010 at 9:10

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
this behavior was already known, but postponed until a solution to handle it 
more gracefully but without losing info it's found.

I'll try to solve it getting the message from the exception but showing it 
without the stack.

```

Original comment by `flik...@gmail.com` on 9 Jul 2010 at 6:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `flik...@gmail.com` on 9 Jul 2010 at 6:28
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `flik...@gmail.com` on 9 Jul 2010 at 6:28
- Changed title: **[findrepe]FileNotFoundException causing FindRepe to exit**

**Comment by GoogleCodeExporter** [comment]
Original comment by `flik...@gmail.com` on 9 Jul 2010 at 6:29
- Changed title: **[findrepe] FileNotFoundException causing FindRepe to exit**

**Comment by GoogleCodeExporter** [comment]
```
r191
```

Original comment by `flik...@gmail.com` on 15 Jul 2010 at 5:03
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
could you verify it has been fixed in version 0.9.0.rc3?

you  can download it at

http://softenido.googlecode.com/files/findrepe-0.9.0.rc3.zip

Now "verbose" option governs the way errors are displayed. Verbose levels have 
their equivalence in java logging levels. default is SEVERE -v is WARNING and 
so on.
```

Original comment by `flik...@gmail.com` on 15 Jul 2010 at 5:12

**Comment by GoogleCodeExporter** [comment]
```
version 0.9.0.rc3 has a bug, it's prone to deadlock, working on it.
```

Original comment by `flik...@gmail.com` on 16 Jul 2010 at 7:47

**Comment by GoogleCodeExporter** [comment]
```
deadlock has been fixed in version 0.9.0.rc4

http://softenido.googlecode.com/files/findrepe-0.9.0.rc4.zip

please, let me know what do you think about how it handles exceptions now

you can increase verbosity using -v -vv -vvv, ...

thank you for the feedback
```

Original comment by `flik...@gmail.com` on 17 Jul 2010 at 9:26

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 0.9.0
```

Original comment by `flik...@gmail.com` on 22 Jul 2010 at 9:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `flik...@gmail.com` on 22 Jul 2010 at 9:11
- Changed state: **Verified**
